evalml/tests/model_understanding_tests/test_feature_explanations.py

```diff
@@ -0,0 +1,405 @@
+                "zero influence",
+            ],
+            "importance": [0.6, -0.2, 0.1, 0.0],
+        }
+    )
+    heavy, somewhat, negative = get_influential_features(
+        importance_df, linear_importance=True
+    )
+    assert heavy == ["heavy influence", "negative influence"]
+    assert somewhat == ["somewhat influence"]
+    assert negative == []
+
+
+def test_get_influential_features_on_boundaries():
+    importance_df = pd.DataFrame(
+        {
+            "feature": ["heavy 1", "heavy 2", "heavy 3", "somewhat 1", "somewhat 2"],
+            "importance": [0.5, 0.2, 0.2, 0.05, 0.05],
+        }
+    )
+    heavy, somewhat, negative = get_influential_features(importance_df)
+    assert heavy == ["heavy 1", "heavy 2", "heavy 3"]
+    assert somewhat == ["somewhat 1", "somewhat 2"]
+    assert negative == []
+
+
+def test_readable_explanation_not_fitted(elasticnet_component_graph):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+
+    with pytest.raises(
+        ValueError,
+        match="Pipelines must be fitted in order to run feature explanations",
+    ):
+        readable_explanation(pipeline)
+
+    with pytest.raises(
+        ValueError,
+        match="Pipelines must be fitted in order to run feature explanations",
+    ):
+        readable_explanation(pipeline, importance_method="feature")
+
+
+def test_readable_explanation_missing_X_y(elasticnet_component_graph, fraud_100):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+    X, y = fraud_100
+    pipeline._is_fitted = True
+
+    with pytest.raises(
+        ValueError, match="required parameters for explaining pipelines"
+    ):
+        readable_explanation(pipeline)
+
+    with pytest.raises(
+        ValueError, match="required parameters for explaining pipelines"
+    ):
+        readable_explanation(pipeline, X)
+
+    with pytest.raises(
+        ValueError, match="required parameters for explaining pipelines"
+    ):
+        readable_explanation(pipeline, y=y)
+
+
+def test_readable_explanation_invalid_importance(elasticnet_component_graph):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+    pipeline._is_fitted = True
+
+    with pytest.raises(ValueError, match="Unknown importance method"):
+        readable_explanation(pipeline, importance_method="fake")
+
+
+def test_readable_explanation_invalid_min_threshold(elasticnet_component_graph):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+    pipeline._is_fitted = True
+
+    with pytest.raises(
+        ValueError, match="minimum importance threshold must be a percentage"
+    ):
+        readable_explanation(pipeline, min_importance_threshold=-1)
+
+    with pytest.raises(
+        ValueError, match="minimum importance threshold must be a percentage"
+    ):
+        readable_explanation(pipeline, min_importance_threshold=2)
+
+
+def test_readable_explanation_permutation(
```

Comment:
In general, I think we could mock `calculate_permutation_importance` in more of these tests; `calculate_permutation_importance` can be costly (though less obvious here since we're using a smaller section of the fraud dataset), and we're not testing whether or not that works here, so we shouldn't need it :)